src/helpers/generate.pagination.details.js

```diff
@@ -0,0 +1,5 @@
+export default (count, rows, offset, limit) => ({
+  pages: Math.ceil(count / limit),
+  currentPage: Math.floor(offset / limit) + 1,
```

Comment:
@nkpremices we probably want to return `count` here too.

Reply:
Alright @MemunaHaruna . I'll add it